GPU/TPCFastTransformation/TPCFastTransform.cxx

```diff
@@ -149,3 +146,62 @@ void TPCFastTransform::print() const
   mDistortion.print();
 #endif
 }
+
+#if !defined(GPUCA_GPUCODE) && !defined(GPUCA_STANDALONE)
```

Comment:
Could you please add ` && !defined(GPUCA_ALIROOT_LIB)`, otherwise this will break AliRoot compilation once we sync GPU to AliRoot.

Reply:
Hi @davidrohr,
I actually tried the code with AliRoot, it compiles and works. 
May be we keep the write/load functionality also for AliRoot, why not? 
It might be useful. 

To test it with Aliroot, I took the AliRoot master, then I copied O2/GPU/TPCFastTransformation folder over, plus some new header file form O2/GPU/Common. The compilation was ok.